traffic_ops/bin/traffic_ops_ort.pl

```diff
@@ -303,12 +303,12 @@ sub revalidate_while_sleeping {
 }
 
 sub os_version {
-  my $release = "UNKNOWN";
-  if (`uname -r` =~ m/.+(el\d)(?:\.\w+)*\.x86_64/)  {
-    $release = uc $1;
-  }
-  exists $supported_el_release{$release} ? return $release
-      : die("unsupported el_version: $release");
+	my $release = "UNKNOWN";
+	if (`uname -r` =~ m/.+(el\d)(?:\.\w+)*\.x86_64/)  {
+		$release = uc $1;
+	}
+	exists $supported_el_release{$release} ? return $release
+		: die("unsupported el_version: $release");
```

Comment:
First let me say I'm a huge fan of turning space-based indentation into tab-based indentation. Just be careful that when you're doing that you aren't turning _alignment_ into _indentation_. This should be indented one level with a tab character then aligned using spaces so that it displays consistently regardless of where your editor places its tabstops.